cdap-api/src/main/java/co/cask/cdap/api/flow/FlowletDefinition.java

```diff
@@ -55,8 +57,14 @@
   private Map<String, Set<Schema>> inputs;
   private Map<String, Set<Schema>> outputs;
 
-  FlowletDefinition(String flowletName, Flowlet flowlet, int instances) {
+  public FlowletDefinition(String flowletName, Flowlet flowlet, int instances) {
```

Comment:
We need to create FlowletDefinition (co.cask.cdap.api.flow) in DefaultFlowConfigurer (co.cask.cdap.internal.app.runtime.flow) which is not in the same package.